### PR TITLE
closes #33 fix test for new design

### LINKED ISSUE #33: Fix Guest can view home page test
Old  test had previous design in mind and some of those links no longer exist

### PR DESCRIPTION
**Pull request by danjwinter**
Make home page dynamic to account for any categories in database
closes #33

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class HomeController < ApplicationController
   def index
+    @category_names = Category.all.pluck(:name)
```
Comment (ToniRib) [inline app/controllers/home_controller.rb]: This comment is for this line and the similar line in the test file: You don't actually need to call "all" here, `Category.pluck` will work just as well.

However, these should be actual links to the categories, so I'm going to try to make that update.

**Comment by ToniRib** [comment]
Test is still passing but now we are actually using jQuery to dynamically link to the category show pages.

File: app/assets/stylesheets/_custom.scss [hunk 2]
```diff
@@ -110,7 +115,7 @@ button.stripe-button-el span {
 
 .job-partial {
   border-bottom: 0.5px solid $black;
-  margin: 0 70px 40px 0;
+  margin: 0 0 40px 0;
```
Comment (ToniRib) [inline app/assets/stylesheets/_custom.scss:118]: I'll make this update later.

**Comment by jwperry** [comment]
Looks solid!

**Pull request merged**